NEWS

```diff
@@ -1,3 +1,36 @@
-Changes in version 0.99.0 (2021-09-29)
+# Changes (2022-05-30)
```

Comment:
Check the example format for NEWS file [here](http://contributions.bioconductor.org/news.html) and stick to that format.  This is to ensure machine-readability and compatibility with automated rendering systems. We may be contacted to correct this otherwise.